pkg/build/generator/generator_test.go

```diff
@@ -370,6 +370,38 @@ func TestInstantiateWithImageTrigger(t *testing.T) {
 				bc = buildConfig
 				return nil
 			}
+		client.GetImageStreamFunc =
+			func(ctx kapi.Context, name string) (*imageapi.ImageStream, error) {
+				return &imageapi.ImageStream{
+					ObjectMeta: kapi.ObjectMeta{Name: name},
+					Status: imageapi.ImageStreamStatus{
+						DockerImageRepository: originalImage,
+						Tags: map[string]imageapi.TagEventList{
+							"tag1": {
```

Comment:
@bparees this looks ugly but I haven't found a better way to inject the tag the trigger expects to change :-)